test/michael_scott_queue/qcheck_michael_scott_queue.ml

```diff
@@ -1,4 +1,4 @@
-open Saturn_lockfree.Queue
+module Queue = Saturn_lockfree.Queue
```

Comment:
Let's not use `open` on the data structure we are about to test.  `open`ing the data structure modules makes it more difficult to identify direct uses of the data structure, which is pretty important when reading these tests.